Kitodo/src/main/java/org/kitodo/production/forms/createprocess/CreateProcessForm.java

```diff
@@ -320,6 +321,18 @@ public String createNewProcess() {
         return this.stayOnCurrentPage;
     }
 
+    private void checkForAutomaticTask() {
```

Comment:
Suggest to rename something like "runAutomaticTask" because in method you trigger more than a check. With this method name, I expect to return a boolean if something is valid.

Reply:
thx, i renamed the method